src/app/screens/BookDetail/components/BookDetail/styles.js

```diff
@@ -0,0 +1,71 @@
+import { StyleSheet } from 'react-native';
+import colors from '../../../../../constants/colors';
+
+export default StyleSheet.create({
+  cover: {
+    height: 105,
+    width: 69,
+    maxHeight: '100%',
```

Comment:
mmmmmmmm?

Reply:
It's ok not? I mean... the "Max" of everything is always 100%... I'm Right?! :rofl: